README.md

```diff
@@ -8,21 +8,32 @@ Please refer to [AppInsights-Home](https://github.com/Microsoft/ApplicationInsig
 
 Prerequisites
 -------------
-1. Java SDK 1.6 or higher
+1.  Java SDK 1.6 or higher
 
 Getting started
 ---------------
-1. Set JAVA_HOME environment variable to point to point to the [JDK installation directory]
-2. Recommended IDE is IntelliJ idea by using ‘Import Project’ rather than ‘Open Project’.
-3. You can use gradlew.bat on Windows to get gradle installed.
+1.  Set JAVA_HOME environment variable to point to point to the [JDK installation directory]
+2.  Recommended IDE is IntelliJ idea by using ‘Import Project’ rather than ‘Open Project’.
+3.  You can use gradlew.bat on Windows to get gradle installed.
```

Comment:
Actually this talks about the standard gradle wrapper so I think it would be better to replace it with:
> To build run `./gradlew build` on Linux systems or `gradlew.bat build` on Windows systems